assets/Styles/styles.css

```diff
@@ -44,3 +53,8 @@
         background-color: rgba(0, 16, 38, 1) !important;
     }
 }
+
+.eterna-input::placeholder {
+    /* If you update this, make sure to also change it in TextInputObject.ts */
```

Comment:
Let's file an issue to make these two colors linked explicitly? Long term.

Reply:
The issue here is that there's no way to access pseudo-elements in the DOM. Looks like there are some possible workaround I wasn't aware of though

Reply:
See #436